deploy.env

```diff
@@ -0,0 +1,10 @@
+# ENV VARIABLES FOR DEPLOYMENT
+# You have to define the values in {}
+APP_NAME=translator
```

Comment:
This must be consistent with a module's name, so generic-translator

Reply:
updated